SetupBLT.cmake

```diff
@@ -41,10 +41,10 @@ if (NOT BLT_LOADED)
     endif()
     unset(_is_multi_config)
 
-    if(${CMAKE_VERSION} VERSION_LESS 3.8.0)
+    if(${CMAKE_VERSION} VERSION_LESS 3.15.0)
```

Comment:
The update of the min version should probably also be noted in the Release-notes due to potential changes to cmake policies.  (I get that this is only a warning from our side, but it's stating blt's expectations).